state/block_test.go

```diff
@@ -0,0 +1,70 @@
+package state
+
+import (
+	"math/big"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/core/types"
+)
+
+func TestGetBlockByNumber(t *testing.T) {
+	dataDir := "../test_data/block"
+
+	// Create & start a new State service
+	stateService := NewService(dataDir)
+	stateService.Start()
+
+	// Close the service, and remove dataDir once test is done
+	defer stateService.Stop()
+	defer func() {
+		if err := os.RemoveAll("../test_data"); err != nil {
```

Comment:
Should we use the `dataDir` variable for this? ie, `os.RemoveAll(dataDir)`

Reply:
done